pyproject.toml

```
@@ -1,27 +1,30 @@
-[tool.poetry]
+[project]
 name = "dagster-sqlmesh"
 version = "0.7.0"
 description = ""
-authors = ["Reuven Gonzales <reuven@karibalabs.co>"]
-license = "Apache 2.0"
+authors = [
+    {name = "Reuven Gonzales", email = "reuven@karibalabs.co"}
+]
+license = {text = "Apache-2.0"}
 readme = "README.md"
+requires-python = ">=3.11,<3.13"
+dependencies = [
+    "dagster>=1.7.8",
+    "sqlmesh<1.0",
+    "pytest>=8.3.2",
+    "pyarrow>=18.0.0",
+]
 
-[tool.poetry.dependencies]
-python = ">=3.11,<3.13"
-dagster = "^1.7.8"
-sqlmesh = "<1.0"
-pytest = "^8.3.2"
-pyarrow = "^18.0.0"
-
-
-[tool.poetry.group.dev.dependencies]
-ipython = "^8.26.0"
-pdbpp = "^0.10.3"
-dagster-webserver = "^1.8.1"
-ruff = "^0.6.2"
-polars = "^1.5.0"
-dagster-duckdb-polars = "^0.24.2"
+[project.optional-dependencies]
```

```suggestion
[dependency-groups]
```

I wouldn't necessarily want this to ship with the package so let's move this to a dependency group. With this I believe you can do `pip install dagster-sqlmesh[dev]` and these deps will be installed.

---

I think this still works with UV fine actually, so no need to use pip separately (I think). Either way, I have adjusted the change (thanks for letting me know)